opentelemetry-sdk/src/opentelemetry/sdk/_configuration/__init__.py

```diff
@@ -103,6 +116,36 @@
 ]
 
 
+def _load_credential_from_envvar(
+    environment_variable: str,
+) -> Optional[
+    tuple[
+        Literal["credentials", "session"], Union[ChannelCredentials, Session]
+    ]
+]:
+    credential_env = os.getenv(environment_variable)
+    if credential_env:
+        credentials = _import_config_component(
+            credential_env, "opentelemetry_otlp_credential_provider"
```

Comment:
I'm wondering if we have docs for entry points. If not, maybe we should open an issue for it.